recipe/bld.bat

```diff
@@ -1,9 +1,11 @@
 :: There is no `gdal-config` on Windows so we need to hardcode gdal's version.
-set GDAL_VERSION=3.0.1
+set GDAL_VERSION=2.4.3
```

Comment:
@isuruf do you know if there is an env var from `conda_build_config.yaml` that we can use here to se both libgdal 2 and 3?